styles.css

```diff
@@ -0,0 +1,265 @@
+* {
+	box-sizing: border-box; 
+	font-family: 'Urbanist', sans-serif;
+}
+
+/*NAV*/
+
+nav {
+	display: flex;
+	justify-content: space-between;
+	padding: 10px;
+	background-color: #41B3A3;
+	z-index: 6;
+}
+
+.mobile-nav {
+	display: none;
+}
+
+.title {
+	margin: 12px;
+
+}
+
+.nav-links {
+	display: flex;
+	justify-content: space-between;
+	padding: 10px;
+	
+}
+
+a {
+	margin: 10px;
+	padding: 10px;
+	border: 3px black solid;
+	border-radius: 5px;
+	text-decoration: none;
+	color: black ;	
+}
+
+
+
+/*ABOUT ME*/
+
+main {
+	background-color: #E8A87C;
+	z-index: 6;
+}
+
+main section {
+	display: flex;
+}
+
+section {
+	width: 60%;
+	margin: 0 auto;
+	padding: 100px;
+}
+
+.picture {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.picture-me {
+	width: 200px;
+	height: 200px;
+	border-radius: 7px;
+}
+
+article {
+	margin-left: 40px;
+}
+
+main section h4 {
+	font-size: 30px;
+}
+
+main section p {
+	font-size: 20px;
+}
+
+/*Projects*/
+.background-projects {
+	background-color:#C38D9E;
+	z-index: 6;
+}
+.projects {
+	width: 60%;
+}
+.projects div {
+	display: flex;
+	padding: 15px;
+}
+
+.project1,.project3 {
+	margin-bottom: 10px;
+	flex-direction: row-reverse;
+	justify-content: space-around;
+	box-shadow:
+	  0 1.5px 1px rgba(0, 0, 0, 0.085),
+	  0 4.6px 3.4px rgba(0, 0, 0, 0.151),
+	  0 12px 15px rgba(0, 0, 0, 0.3);
+}
+
+.project2 {
+	justify-content: space-around;
+	box-shadow:
+	  0 1.5px 1px rgba(0, 0, 0, 0.085),
+	  0 4.6px 3.4px rgba(0, 0, 0, 0.151),
+	  0 12px 15px rgba(0, 0, 0, 0.3);
+}
+
+.prj-img{
+	margin-left: 10px;
+	border-radius: 7px;
+}
+.prj2img {
+	margin-right: 10px;
+	border-radius: 7px;
+}
+
+aside {
+	margin-bottom: 10px;
+}
+
+/*Footer*/
+footer {
+	z-index: 6;
+	background-color:#E27D60; 
+}
+footer section{
+	display: flex;
+	justify-content: center;
+	padding: 30px;
+}
+
+.footer-img {
+	width: 25px;
+	height: 25px;
+}
+/*Button*/
+
+.totop {
+	position: fixed;
+	bottom: 7%;
+	right: 3%;
+	border: none;
+}
+
+.topimg {
+	width: 30px;
+	height: 30px;
+}
+
+/*Media queries mid screen*/
+@media screen and (min-width: 760px) and (max-width: 1100px) {
+/*	ABOUT*/
+	main section {
+		display: flex;
+		width: 70%;
+		padding: 5px;
+	}
+	
+	.picture-me {
+		width: 150px;
+		height:150px;
+	}
+
+/*PROJECTS*/
+	.projects {
+		width: 80%;
+		padding: 30px;
+	}
+	.projects div {
+		display: flex;
```

Comment:
this is already being declared higher up the stylesheet so doesnt need to happen again.